src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -103,6 +102,12 @@ public ArgumentMultimap tokenize() throws ParseException {
         return (prefixIndex == -1
                 ? Optional.empty()
                 : Optional.of(new PrefixPosition(prefix, prefixIndex + 1)));
+    }*/
+
+    private static int findPrefixPosition(String argsString, String prefix, int fromIndex) {
+        int prefixIndex = argsString.indexOf(" " + prefix, fromIndex);
+        return prefixIndex == -1 ? -1
+                : prefixIndex + 1; // +1 as offset for whitespace
```

Comment:
Maybe we should re implement PrefixPosition in the future.

Reply:
Thanks for the feedback Joshua, but this was actually the original implementation for AB3.